### ISSUE TITLE
Background and foreground colors do not have a sufficient contrast ratio

### ISSUE THREAD

**Issue by lakshyatyagi24** [StudentCode-in/StudentCode-in.github.io#112]
Fix  any of the following:   Element has insufficient color contrast of 2.5 (foreground color:  #ffffff, background color: #1bb1dc, font size: 9.8pt (13px), font  weight: normal). Expected contrast ratio of 4.5:1
<a href="#about" class="btn-get-started scrollto">
--
Fix  any of the following:   Element has insufficient color contrast of 3.97 (foreground color:  #ffffff, background color: #007bff, font size: 15.0pt (20px), font  weight: normal). Expected contrast ratio of 4.5:1
<a  class="btn btn-primary btn-lg sp-btn"  href="https://docs.google.com/forms/d/1EvhKxub2kroqk4s8KlBpmLwXE4vTc3-wy5Xa8qK5Y…"  role="button">
--
Fix  any of the following:   Element has insufficient color contrast of 2.35 (foreground color:  #1bb1dc, background color: #f5f8fd, font size: 10.5pt (14px), font  weight: normal). Expected contrast ratio of 4.5:1
<a href="mailto:<nowiki>

**Comment by souravseal99** [comment]
Hi there, please let me know if I can work on this ?

**Comment by lakshyatyagi24** [comment]
sure @souravseal99

**Comment by souravseal99** [comment]
> sure @souravseal99

ok, I am working on it.

**Comment by bhaveshsonje** [comment]
Hey, I  am Bhavesh Sonje. How can I contribute to this open source project for my open source class? Please assign me a task.
Contact Me: sonjebhavesh623@gmail.com
GitHub: bhaveshsonje

**Comment by Abhishekcuso** [comment]
I would like to solve this issue

**Comment by Abhishekcuso** [comment]
Hey, I am Abhishek Khairnar . How can I contribute to this open source project for my open source class? Please assign me a task.
Contact Me: [khairnarabma19ie@student.mes.ac.in](mailto:khairnarabma19ie@student.mes.ac.in)
GitHub: Abhishekcuso

**Comment by lakshyatyagi24** [comment]
@Abhishekcuso You can solve this you can create PR. I will review the same and merge.

**Comment by VinayKalyani0** [comment]
Hey, I am Vinay Kalyani. How can I contribute to this open source project for my open source class? Please assign me a task.
Contact Me: 2019vinay.kalyani@ves.ac.in
Github: VinayKalyani0

**Comment by lakshyatyagi24** [comment]
@VinayKalyani0 You can solve any issue in this repository can create PR. I will review the same and merge.

You can also pick any project from my profile too.

**Comment by Vishalsuthar88** [comment]
I know how to fix this issue, please assigned this task to me.

**Comment by lakshyatyagi24** [comment]
@Vishalsuthar88 assigned it to you
